helper-cli/src/main/kotlin/commands/RemoveConfigurationEntriesCommand.kt

```diff
@@ -110,7 +110,7 @@ internal class RemoveConfigurationEntriesCommand : CommandWithHelp() {
             appendln("  path excludes: $removedPathExcludes")
             appendln("  scope excludes: $removedScopeExcludes")
             appendln("  rule violation resolutions: $removedRuleViolationResolutions")
-        }.let{ println(it) }
+        }.let { println(it) }
```

Comment:
Nit: This formatting fix probably belongs to another commit.

Reply:
Done.